README.md

```diff
@@ -234,6 +234,31 @@ const tenantGate = new CustomTenantGate();
 const dwn = await Dwn.create({ messageStore, dataStore, eventLog, tenantGate });
 ```
 
+### Custom Signature Signer
+Instead of using `PrivateKeySigner` to perform messger signing when you have the private key readily available, you can implement a customer signer to interface with external signing service, API, HSM, TPM etc and use it for signing your DWN messages:
```

Comment:
```suggestion
Instead of using `PrivateKeySigner` to perform message signing when you have the private key readily available, you can implement a customer signer to interface with external signing service, API, HSM, TPM etc and use it for signing your DWN messages:
```

Reply:
Since we don't mention `PrivateKeySigner` elsewhere in the README, maybe phrasing it like:
"If you have the private key readily available, it is recommended to use the built-in `PrivateKeySigner`. Otherwise, you can implement a custom signer ..."

Reply:
Good suggestion! Done. Orthogonally, will working on modifying the `Signer` interface as we discussed.